src/Components/Facility/DischargeModal.tsx

```diff
@@ -210,9 +210,12 @@ const DischargeModal = ({
       title={
         <div>
           <p>Discharge patient from CARE</p>
-          <span className="mt-1 flex gap-1 text-sm font-medium text-warning-500">
+          <span className="mt-1 flex gap-1 text-sm font-medium text-danger-500">
             <CareIcon icon="l-exclamation-triangle" className="text-base" />
-            <p>Caution: this action is irreversible.</p>
+            <p>
+              Caution: Once a patient is marked as expired, the patient file
+              cannot be transferred or edited. Please proceed with caution.
+            </p>
```

Comment:
This caution can be shown when the selected reason is expired right? The original caution notice ("this action is irreversible") still holds true for non-expired situations too right.